ID.me WebVerify SDK/IDmeWebVerify.m

```diff
@@ -643,7 +642,7 @@ - (void)makePostRequestWithUrl:(NSString *_Nonnull)urlString parameters:(NSStrin
 }
 
 #pragma mark - Keychain access
-- (void)saveDataFromJson:(NSDictionary* _Nonnull)json {
+- (void)saveKeychainDataFromJson:(NSDictionary* _Nonnull)json scope:(NSString* _Nonnull)scope {
```

Comment:
I was thinking something like `saveTokenDataFromJson`, I think it's more clear, if you read it then you will know that this functions expect a JSON representation of a token object (access token + refresh token + etc.) What do you think?